index.html

```diff
@@ -24,7 +24,7 @@
     </header>
 
     <section class="headline">
-      <h1 class="headline__title">I’m Ahmed Glad to see you!</h1>
+      <h1 class="headline__title">I’m Ahmed H. Glad to see you!</h1>
       <p class="headline__text">
         I’m a software developer! I can help you build a product , feature or
         website Look through some of my work and experience! If you like what
```

Comment:
- [ ] Let's fix the background image of the headline section as the background image is not displaying on the page. Please consider adding the background image so as to match the Figma template.

Your page:
<img width="946" alt="Loom_aO5A0uJIIv" src="https://user-images.githubusercontent.com/81468675/149226099-f366d711-c31f-48b7-9e5b-267854d027a6.png">